concepts/Common/Security Considerations/40 Content Security Policy/15 CSP Source List.md

```diff
@@ -0,0 +1,74 @@
+    </tr>
+    <tr>
+        <td><code>'self'</code></td>
+        <td><code>script-src 'self'</code></td>
+        <td>Allows a web site to load resources from the same origin (same scheme, host and port).</td>
+    </tr>
+    <tr>
+        <td><code>data:</code></td>
+        <td><code>img-src 'self' data:</code></td>
+        <td>Allows a web site to load resources via the data scheme (for example, Base64 encoded images).</td>
+    </tr>
+    <tr>
+        <td><code>domain.example.com</code></td>
+        <td><code>img-src domain. example.com</code></td>
+        <td>Allows a web site to load resources from the specified domain name.</td>
+    </tr>
+    <tr>
+        <td><code><em>*.example.com</em></code></td>
+        <td><code>img-src *.example.com</code></td>
+        <td>Allows a web site to load resources from any subdomain under <code>example.com</code>.</td>
+    </tr>
+    <tr>
+        <td><code><em>https://cdn.com</em></code></td>
+        <td><code>img-src https://cdn.com</code></td>
+        <td>Allows a web site to load resources only over HTTPS matching the given domain.</td>
+    </tr>
+    <tr>
+        <td><code>https:</code></td>
+        <td><code>img-src https:</code></td>
+        <td>Allows a web site to load resources only over HTTPS on any domain.</td>
+    </tr>
+    <tr>
+        <td><code>'unsafe-inline'</code></td>
+        <td><code>script-src 'unsafe-inline'</code></td>
+        <td>Allows a web site to use inline source elements such as style attribute, <code>onclick</code>, or script tag bodies (depends on the context of the source it is applied to) and <code>javascript:</code> URIs.</td>
+    </tr>
+    <tr>
+        <td><code>'unsafe-eval'</code></td>
+        <td><code>script-src 'unsafe-eval'</code></td>
+        <td>Allows a web site to perform unsafe dynamic code evaluation such as JavaScript <code>eval()</code>.</td>
+    </tr>
+    <tr>
+        <td><code>'nonce-'</code></td>
+        <td><code>script-src 'nonce-rAnd0m'</code></td>
+        <td>Allows an inline script or CSS to execute if the script (for example, <code>&lt;script nonce="rAnd0m"&gt;</code>) tag contains a nonce attribute matching the nonce specifed in the CSP header. The nonce should be a secure random string, and should not be reused. CSP Level 2 </td>
+    </tr>
+    <tr>
+        <td><code>'strict-dynamic'</code></td>
+        <td><code>script-src 'strict-dynamic'</code></td>
+        <td>Enables an allowed script to load additional scripts via non-"parser-inserted" script elements (for example, <code>document.createElement('script');</code> is allowed). CSP Level 3</td>
+    </tr>
+    <tr>
+        <td><code>'unsafe-hashes'</code></td>
+        <td><code>script-src 'unsafe-hashes'</code></td>
+        <td>Allows you to enable scripts in event handlers (for example, <code>onclick</code>). Does not apply to <code>javascript:</code> or inline <code>&lt;script&gt;</code> CSP Level 3 </td>
```

Comment:
Judging by the other items, you're probably missing punctuation here. 

```suggestion
        <td>Allows you to enable scripts in event handlers (for example, <code>onclick</code>). Does not apply to <code>javascript:</code> or inline <code>&lt;script&gt;</code>. CSP Level 3 </td>
```

Reply:
I think so